backend/src/index.ts

```diff
@@ -136,3 +136,29 @@ app.use("/api/volunteer-types", volunteerTypeRoutes);
 // });
 /** Qualification Routes */
 app.use("/api/qualifications", qualificationRoutes);
+
+//Only Start The Sever If Mongoose Is Connected
+/*const StartServer = async () => {
```

Comment:
This is all commented out?

Reply:
i will remove the commented-out code since it was an earlier attempt when I was learning about nodemailer